HHS/Head-Start-TTADP
# [TTAHUB-1069] Add AR App Loader

## Description of change

When creating an AR we want to put a loader over the entire application.

## How to test

We should display a App loader when performing the following AR actions.
- 'Save and Continue'
- 'Save Draft'
- 'Save Goal' 
- Jumping between AR Pages via NAV

It should NOT occur during an auto save.

The AR creation process should work as it did before.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1069


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
